client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -158,7 +156,11 @@ class DesignTypeWithStoreStep extends Component {
 		return (
 			<div className="design-type-with-store__substep-wrapper">
 				<div className={ storeWrapperClassName }>
-					{ this.renderStoreStep() }
+					<PressableStoreStep
+						{ ... this.props }
+						onBackClick={ this.handleStoreBackClick }
+						setRef={ this.setPressableStore }
+					/>;
```

Comment:
There's an extra `;` here that gets rendered. 

![screen shot 2017-03-17 at 13 02 56](https://cloud.githubusercontent.com/assets/184938/24040480/1233fba4-0b12-11e7-94d3-b291ec8c6808.png)